### PR TITLE
.ci/deps.opam.sh: Fix uncached opam dependencies

### PR DESCRIPTION
**Pull request by harshhx17**
This makes infer use yojson v1.3.3 and biniou v1.0.6

Closes https://github.com/coala/coala-bears/issues/2006

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by harshhx17** [comment]
@jayvdb 
Please review

**Comment by harshhx17** [comment]
That error is unrelated...
I will just rebuild :wink:

**Comment by harshhx17** [comment]
Again the error is not related...
git repository not found :expressionless: 
Rebuilding...

File: .ci/deps.opam.sh [hunk 1]
```diff
@@ -11,6 +11,10 @@ if [ ! -e ~/infer-linux64-v0.7.0/infer/bin ]; then
   opam init --y
   opam update
 
+  opam pin add --yes --no-action yojson 1.3.3
```
Comment (jayvdb) [inline .ci/deps.opam.sh]: Add `# See <URL>` above this line, like I did below, and put the two commands together as they are part of the same fix.

Comment (harshhx17) [reply .ci/deps.opam.sh]: Ok

**Comment by harshhx17** [comment]
Done @jayvdb

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2451/commits/3f0f903cc6fe2c6970376eb9d6393d37022d60f3

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**